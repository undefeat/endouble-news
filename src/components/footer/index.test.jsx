import React from 'react';
import shallow from 'react-test-renderer/shallow';
import Footer from '.';

describe('Footer', () => {
    it('should correctly render', () => {
        const tree = shallow.createRenderer().render(<Footer />);

        expect(tree).toMatchSnapshot();
    });
});
