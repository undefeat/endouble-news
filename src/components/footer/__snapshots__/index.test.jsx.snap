// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should correctly render 1`] = `
<footer
  className="footer"
>
  <span>
    Powered by
  </span>
  <a
    className="footer__link"
    href="https://newsapi.org"
    rel="noopener noreferrer"
    target="_blank"
  >
    NewsAPI.org
  </a>
</footer>
`;
